packages/dev/sharedUiComponents/src/nodeGraphSystem/automaticProperties.ts

```diff
@@ -0,0 +1,38 @@
+import type { IEditablePropertyOption } from "core/Decorators/nodeDecorator";
+import type { StateManager } from "./stateManager";
+
+/**
+ * Function used to force a rebuild of the node system
+ * @param source source object
+ * @param stateManager defines the state manager to use
+ * @param propertyName name of the property that has been changed
+ * @param notifiers list of notifiers to use
+ * @param forceUpdate defines if we should force a react update
+ */
+export function ForceRebuild(source: any, stateManager: StateManager, propertyName: string, notifiers?: IEditablePropertyOption["notifiers"], forceUpdate = false) {

```

Comment:
nit: if no other exported functions are going to be in this file, could the file be named after its function (ForceRebuild)?

Reply:
Good take but I plan to add more ;)